src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -1660,7 +1660,7 @@ static void uct_ib_mlx5_devx_check_odp(uct_ib_mlx5_md_t *md,
                 capability.odp_cap.memory_page_fault_scheme_cap);
         version = 2;
     } else {
-        if (md_config->devx_objs & UCS_BIT(UCT_IB_DEVX_OBJ_RCQP)) {
+        if (md_config->devx_objs != 0) {
```

Comment:
need to check RCQP|DCQP
other DevX objects should work with ODPv1

Reply:
formally its correct, but verbs QP can't point to devx SRQ, so SRQ also should be verbs

Reply:
i'd prefer a direct check, there are other devx objects that may not be relevant for odp